bigtable/tests/unit/test_table.py

```diff
@@ -16,6 +16,53 @@
 import unittest
 
 
+class Test___mutate_rows_request(unittest.TestCase):
+
+    def _call_fut(self, table_name, rows):
+        from google.cloud.bigtable.table import _mutate_rows_request
+
+        return _mutate_rows_request(table_name, rows)
+
+    def test__mutate_rows_too_many_mutations(self):
+        from google.cloud.bigtable.table import TooManyMutationsError
+        from google.cloud.bigtable.row import DirectRow
+        from google.cloud.bigtable._generated.data_pb2 import Mutation
+
+        mutation = Mutation()
+        rows = [DirectRow(row_key=b'row_key', table='table'),
+                DirectRow(row_key=b'row_key_2', table='table')]
+        rows[0]._pb_mutations = [mutation for _ in range(0, 50000)]
+        rows[1]._pb_mutations = [mutation for _ in range(0, 50001)]
```

Comment:
Here is where I was suggesting just using fewer mutations by mocking out the threshold.